### ISSUE TITLE
Does adjustment of lr work?

### ISSUE THREAD

**Issue by windelvin** [zsyOAOA/DifFace#14]
Have you ever tried to decrease the lr(the default value is 1e-4) for the training procedure of diffusion model?
Since the loss converges very quick(less than 1W iters), but the lr keeps constant during the whole training time.

**Comment by zsyOAOA** [comment]
No, the learning rate is fixed as 1e-4 following the setting in DDPM.

**Comment by zsyOAOA** [comment]
We have not trained the model with 1024 x 1024, which requires huge computation resources.
